### DISCUSSION TITLE
Free trade

### DISCUSSION THREAD

**Discussion by BloodSin666** [FakeFishGames/Barotrauma#5478] [Ideas and suggestions]
Hey, guys.
I use Google translator, so if you don't understand something, ask again. I will try to answer more correctly.
How do you like the idea?
All crew members can collect what they need from the merchant.
And the captain of the submarine is already deciding what to buy.
The captain can also adjust the purchases of the crew members.

**Comment by Xeavas** [comment]
I think I can try to infer what you're trying to say.

1. A wallet system to allocate allowances to crew to let them purchase their own things (with settings to blacklist certain items or via point 2)
2. An option to enable audit by the host/captain to allow purchases (to prevent buying things that are potentially harmful/not to the host's/captain's liking)
3. Potential system to allow the crew to suggest things for the host/captain to buy, a simple "I would like this item" by the side with the name of the person suggesting

If what im getting is correct, I would like to support this idea

**Reply by BloodSin666** [reply]
> I think I can try to infer what you're trying to say.
> 
> 1. A wallet system to allocate allowances to crew to let them purchase their own things (with settings to blacklist certain items or via point 2)
> 2. An option to enable audit by the host/captain to allow purchases (to prevent buying things that are potentially harmful/not to the host's/captain's liking)
> 3. Potential system to allow the crew to suggest things for the host/captain to buy, a simple "I would like this item" by the side with the name of the person suggesting
> 
> If what im getting is correct, I would like to support this idea

Yes
